managers/tests/test_views.py

```diff
@@ -258,119 +258,85 @@ def test_delete_project_function_view_should_not_delete_project_on_non_admin_req
         self.assertEqual(Project.objects.all().count(), 1)
 
 
-class ManageTaskActivitiesInProjectViewTests(TestCase):
+class ManageTaskActivitiesTests(TestCase):
     def setUp(self):
         super().setUp()
         self.user_admin = AdminUserFactory()
         self.client.force_login(self.user_admin)
-        self.default_task_activity_1 = TaskActivityTypeFactory(is_default=True)
-        self.default_task_activity_2 = TaskActivityTypeFactory(is_default=True)
-        self.not_default_task_activity = TaskActivityTypeFactory()
-
         self.project = ProjectFactory()
-        self.url = reverse("project-task-activities", kwargs={"pk": self.project.pk})
+        self.task_activities = TaskActivityTypeFactory()
```

Comment:
this is `task_activity` not `task_activities`. You're creating here only one instance of it

Reply:
changed